app/src/main/res/values/strings.xml

```diff
@@ -46,10 +46,10 @@
 
     <string name="firstrun_dialog_message">
         DNSSetter lets you change your DNS servers. This is temporary and it will not resist if your
-        IP address changes or if you reboot your device.\n
+        IP address changes or if you reboot your operating system.\n
```

Comment:
Does your app use the bootloader or the recovery for a very strange reason? Or do you prefer to use "device" for avoiding confusion/misunderstanding by introducing a confusion between hardware and software? Or is there another reason?